packages/disml/disml.0.2.4/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "Adelyn Breedlove <tenseiken595@gmail.com>"
+authors: "Adelyn Breedlove <tenseiken595@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/Mishio595/disml"
+doc: "https://mishio595.gitlab.io/disml/"
+dev-repo: "git+https://gitlab.com/Mishio595/disml"
+bug-reports: "https://gitlab.com/Mishio595/disml/issues"
+tags: ["discord"]
+synopsis: "An OCaml library for interfacing with the Discord API"
+description: """
+Dis.ml is a library that provides a high-level interface to the Discord API.
+Key features include:
+* Automatic sharding
+* Deserialization of Discord objects to record types with related helper methods
+* Automatic rate-limiting
+
+For examples, see `/bin` in the git repo.
+"""
+depends: [
+  "ocaml" {>= "4.07.0"}

```

Comment:
is there a reason why you rely on such a recent version of OCaml?

Reply:
I haven't tested with older compiler versions, testing how far back this is compatible currently.